interfaces.go

```diff
@@ -160,6 +160,37 @@ var sensitiveHeaders = map[string]struct{}{
 	"Cookie":              {},
 	"X-Forwarded-For":     {},
 	"X-Real-Ip":           {},
+	"Password":            {},
+	"Passwd":              {},
+	"Secret":              {},
+	"Api-Key":             {},
+	"Apikey":              {},
+	"Auth":                {},
+	"Credentials":         {},
+	"Mysql-Pwd":           {},
+	"Privatekey":          {},
+	"Private-Key":         {},
+	"Token":               {},
+	"Ip-Address":          {},
+	"Session":             {},
+	"Csrftoken":           {},
+	"Sessionid":           {},
+	"Remote-Addr":         {},
+	"X-Csrftoken":         {},
+	"Set-Cookie":          {},
+	"X-Api-Key":           {},
+	"Aiohttp-Session":     {},
+	"Connect.sid":         {},
+	"Csrf-Token":          {},
+	"Csrf":                {},
+	"_csrf":               {},
+	"_csrf_token":         {},
+	"PHPSESSID":           {},
```

Comment:
Why would I need this in a Go app?

Reply:
@cleptric let's keep one conversation here for all these comments.

The headers you're asking about were added by @antonpirker to the python SDK [here](https://github.com/getsentry/sentry-python/pull/1972), and Anton would be best suited to answer this.

If I had to guess the reason, I'd say it is because these keys can appear in any client request/response objects, e.g when we make an http request from the project that uses sentry (the http client) to another app (the http server) that can be running on anything